apps/molgenis-components/src/client/client.ts

```diff
@@ -69,20 +80,21 @@ export default {
         );
         return dataResp[tableId];
       },
-      fetchRowData: async (tableName, rowId) => {
-        const tableId = convertToPascalCase(tableName);
+      fetchRowData: async (tableName: string, rowId: IRow) => {
+        const tableId = convertToPascalCase(tableName) || tableName;
         if (metaData === null) {
           metaData = await fetchMetaData(myAxios, schemaName);
           if (!schemaName) {
             schemaName = metaData.name;
           }
         }
         const tableMetaData = metaData.tables.find(
-          (table) => table.id === tableId && table.externalSchema === schemaName
+          (table: ITableMetaData) =>
+            table.id === tableId && table.externalSchema === schemaName
         );
-        const filter = tableMetaData.columns
-          .filter((column) => column.key === 1)
-          .reduce((accum, column) => {
+        const filter = tableMetaData?.columns
+          ?.filter((column) => column.key === 1)
+          .reduce((accum: any, column) => {
```

Comment:
We need the ? here because [].find might return undefined. Alternatively we could check if the meta data is found and throw and error, but that's a bit much I think (for this pr).